src/style.css

```diff
@@ -0,0 +1,897 @@
+  display: flex;
+  margin-top: 10px;
+  flex-wrap: wrap;
+  width: 100%;
+  align-items: center;
+  justify-content: start;
+}
+/* End home */
+
+.preview-container{
+  width: 250px;
+  height: 160px;
+  border-radius: 8px;
+  margin: 5px;
+}
+
+.preview-component{
+  width: 250px;
+  height: 160px;
+  padding: 1em 0.5em;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: var(--color-regular-text);
+  line-height: 3rem;
+}
+
+.main-component{
+  position: absolute;
+  background-color: var(--color-primary);
+  line-height: 2rem;
+  transition: all 0.3s 0.1s ease;
+}
+
+.overlay-component{
+  color: rgba(0,0,0,0);
+  font-family: "Raleway";
+  font-size: var(--text-large);
+  position: relative;
+  background: none;
+  left: 0.45rem;
+  bottom: 0.45rem;
+  border: 1px solid var(--color-primary-dark);
+  transition: all 0.4s ease;
+}
+
+.preview-container:hover .overlay-component{
+  position: relative;
+  background-image: rgba(250, 250, 250, 0.8);
+  backdrop-filter: blur(7px);
+  color: var(--color-overlay-text);
+}
+
+.preview-container:hover .main-component{
+ color: var(--color-primary); 
+ transition: all 0s;
+}
+/* Collection preview */
+.collection-preview{
+  display: block;
+  position: absolute;
+  background-color: var(--color-primary);
+}
+
+.outline-border{
+  position: relative;
+  background: none;
+  left: 0.45rem;
+  bottom: 0.45rem;
+  border: 1px solid var(--color-primary-dark);
+}
+
+.preview-header{
+  width: 100%;
+  color: var(--color-contrast-text);
+  position: absolute;
+  top: 1em;
+  font-size: 1.2rem;
+  text-align: center;
+}
+
+.preview-details{
+  position: absolute;
+  width: 100%;
+  top: 3em;
+}
+.preview-footer{
+  width: 100%;
+  position: absolute;
+  bottom: 1.5rem;
+  font-size: 0.8rem;
+  text-align: center;
+  color:var(--color-overlay-text);
+
+}
+/* End collection preview */
+
+/* Card preview */
+
+.card-preview-back{
+  display: block;
+  position: absolute;
+  background: var(--color-primary-dark);
+  color: var(--color-regular-text);
+  text-align: center;
+  padding-top: 40px;
+  overflow: hidden;
+}
+.card-preview-front{
+  position: relative;
+  color: var(--color-regular-text);
+  background: var(--color-primary);
+}
+.preview-info{
+  margin: 10px 20px;
+}
+
+.preview-container:hover .card-preview-front{
+  transform: translateY(80px) ;
+  transition: all 300ms ease;
+  box-shadow: var(--overlay-shadow);
+}
+.overlay{
+  overflow: hidden;
+}
+/* End card preview */
+
+.link-preview-title{
+  font-size: var(--text-medium);
+  line-height: inherit;
+}
+
+.link-preview-details{
+  font-size: var(--text-small);
+  line-height: inherit;
+}
+
+
+.tabbar li{
+  display: inline;
+  margin-left: 2vw;
+  text-decoration: none;
+  font-size: var(--text-medium);
+  cursor: pointer;
+}
+
+
+.tabbar{
+  margin-top: 1vw;
+  width: 800px;
+}
+
+hr{
+  margin-top: 0.4em;
+  border: 0.5px solid var(--color-primary-dark);
```

Comment:
Non-integer pixel values are not really a good choice, because there is no such thing as a standard which prescribes browsers to treat this case specifically. This may result in different layout on different browsers.